### PR TITLE
Add org.americanmonk.TipitakaPaliReadder

### PR DESCRIPTION
**Pull request by bksubhuti**
### Please confirm your submission meets all the criteria

- [x ] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [ x] My pull request follows the instructions at [App Submission][submission].
- [ x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [ x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

I hope I did everything correctly.  I'm not sure about the tags.. but this is pretty stable.  I tested locally with a local archive and also tested remotely with the remote github archive and sha256.  Not sure about the tags or archive name.  This app is routinely updated on ios macos windows android and appimage.  So the app has been in production for a long time.  2300 active users in android alone.  However, I only expect a handful in flatpak since it is so specialized.

### CODE REVIEW

**Comment by hfiguiere** [comment]
submission pr are supposed to be against the `new-pr` branch. not master.

File: org.americanmonk.TipitakaPaliReader.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      - install -Dm644 tpr/icons/32x32/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/32x32/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/icons/48x48/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/48x48/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/icons/128x128/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/128x128/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/icons/256x256/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/256x256/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/icons/512x512/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/512x512/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/org.americanmonk.TipitakaPaliReader.desktop /app/share/applications/org.americanmonk.TipitakaPaliReader.desktop
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
```
Comment (hfiguiere) [inline org.americanmonk.TipitakaPaliReader.yml:34]: this doesn't prevent building an aarch64 flatpak.

**Comment by hfiguiere** [comment]
bot, build org.americanmonk.TipitakaPaliReader

File: org.americanmonk.TipitakaPaliReader.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      - ln -s /usr/lib/x86_64-linux-gnu/libsqlite3.so.0 /app/lib/libsqlite3.so
+      - install -Dm644 tpr/icons/24x24/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/24x24/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/icons/32x32/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/32x32/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/icons/48x48/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/48x48/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/icons/128x128/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/128x128/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/icons/256x256/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/256x256/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/icons/512x512/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/512x512/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/org.americanmonk.TipitakaPaliReader.desktop /app/share/applications/org.americanmonk.TipitakaPaliReader.desktop
```
Comment (hfiguiere) [inline org.americanmonk.TipitakaPaliReader.yml:30]: I don't see the mandatory appstream file being installed here.

**Comment by bksubhuti** [comment]
If I ask to do the pr on the new-pr.. it does not let me do.. "nothing to compare"

File: org.americanmonk.TipitakaPaliReader.yml (hunk 2 above)
Comment (bksubhuti) [reply org.americanmonk.TipitakaPaliReader.yml:30]: what is mandatory app stream?

Comment (hfiguiere) [reply org.americanmonk.TipitakaPaliReader.yml:30]: The relevant part in the documentation you acknowledged having read:

https://docs.flathub.org/docs/for-app-authors/requirements/#appstream

**Pull request closed without merging**